tools/scripts/generate_binary_build_matrix.py

```diff
@@ -240,6 +240,10 @@ def get_libtorch_install_command(
         build_name = f"libtorch-macos-{arch}-latest.zip"
         if channel == RELEASE:
             build_name = f"libtorch-macos-{mod.CURRENT_VERSION}.zip"
+        # Please note this is temporary, to be removed when doing branch cut for 2.2 release
+        elif channel == TEST:
+            build_name = "libtorch-macos-latest.zip"
```

Comment:
Shouldn't we be testing version for TEST channel as well (Otherwise we are not testing what is about to be released?)